### PR TITLE
Update GeoJsonReaderDriver class: typo and illogical alternatives

### PR DESCRIPTION
**Pull request by gpbp**
I fixed some typo mistakes and those illogical alternatives.

### CODE REVIEW

File: h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java [hunk 1]
```diff
@@ -551,23 +551,15 @@ private void parseMultiPolygonMetadata(JsonParser jp) throws IOException, SQLExc
             jp.nextToken(); // START_ARRAY [ coordinates             
             jp.nextToken(); //Start the polygon
             while (jp.getCurrentToken() != JsonToken.END_ARRAY) {
-                //Parse the polygon
+                //Parses the polygon
                 jp.nextToken(); //Start the RING
                 int linesIndex = 0;
                 while (jp.getCurrentToken() != JsonToken.END_ARRAY) {
-                    if (linesIndex == 0) {
```
Comment (gpbp) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java]: Illogical alternative: if linesIndex == 0 or linesIndex != 0, we do the same thing!

File: h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java [hunk 2]
```diff
@@ -551,23 +551,15 @@ private void parseMultiPolygonMetadata(JsonParser jp) throws IOException, SQLExc
...
-                        parseCoordinatesMetadata(jp);
-                    } else {
-                        parseCoordinatesMetadata(jp);
-                    }
+                    parseCoordinatesMetadata(jp);
                     jp.nextToken();//END RING
                     linesIndex++;
                 }
-                if (linesIndex > 1) {
```
Comment (gpbp) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java]: Another illogical alternatives: if linesIndex > 1 or linesIndex < 1, same thing will happen!

File: h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java [hunk 3]
```diff
@@ -628,15 +620,14 @@ private void parseCoordinateMetadata(JsonParser jp) throws IOException {
         jp.nextToken(); // second value
         //We look for a z value
         jp.nextToken();
-        if (jp.getCurrentToken() == JsonToken.END_ARRAY) {
```
Comment (gpbp) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java]: Malformed alternative: we need just one "if" to complete the alternative.

**Comment by SPalominos** [comment]
Please, link this PR to an issue. If you can't find an existing issue, create one.

File: h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java [hunk 4]
```diff
@@ -362,44 +362,44 @@ private void parseParentGeometryMetadata(JsonParser jp) throws IOException, SQLE
...
      * "geometry":{"type": "Point", "coordinates": [102.0,0.5]}
      *
-     * @param jsParser
+     * @param jp
      * @throws IOException
      */
-    private void parseGeometryMetadata(JsonParser jsParser, String geometryType) throws IOException, SQLException {        
+    private void parseGeometryMetadata(JsonParser jp, String geometryType) throws IOException, SQLException {        
         if (geometryType.equalsIgnoreCase(GeoJsonField.POINT)) {
-             parsePointMetadata(jsParser);
```
Comment (ebocher) [inline h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java]: hum cosmetic change

**Pull request merged**